app/controllers/Contributions.java

```diff
@@ -2244,11 +2245,21 @@ public static Contribution createContribution(Contribution newContrib,
 
         Logger.info("Using Etherpad server at: " + etherpadServerUrl);
         Logger.debug("Using Etherpad API Key: " + etherpadApiKey);
-
-        if (type != null && (type.equals(ContributionTypes.PROPOSAL) || type.equals(ContributionTypes.NOTE))) {
-            ContributionsDelegate.createAssociatedPad(etherpadServerUrl, etherpadApiKey, newContrib, t, containerResourceSpace.getResourceSpaceUuid());
-        }
-
+        
+        Campaign ca = Campaign.readByResourceSpaceId(containerResourceSpace.getResourceSpaceId());        
+        if (ca != null){
+        	List<Config> campaignConfigs = Config.findByCampaignAndKey(ca.getUuid(), "appcivist.campaign.disable-etherpad");        	
+        	for(Config cc: campaignConfigs){
+        		if (cc.getValue().equalsIgnoreCase("FALSE")){
```

Comment:
- [ ] Also if there is no config with that key, we should allow the use of etherpads because the default value for this key is FALSE (you can read the default value for this variable - and any other - from `GlobalConfigKeys.CONFIG_DEFAULTS.get(GlobalDataConfigKeys.APPCIVIST_CAMPAIGN_DISABLE_ETHERPAD)`